okio/src/jvmMain/kotlin/okio/RawSink.kt

```diff
@@ -26,6 +26,9 @@ actual interface RawSink : Closeable, Flushable {
   @Throws(IOException::class)
   actual override fun flush()
 
+  @Suppress("ACTUAL_WITHOUT_EXPECT")
+  actual suspend fun flushSuspend() = flush()
```

Comment:
we also missing the `awaitSpace` operation on `RawSink`

Reply:
Please be more constructive -- what such a method is supposed to do, what is impossible to implement without it and so on. 

A demonstration would be extremely helpful.

Reply:
I am especially interested in a contract of such a method -- for now, there is no notion of "underlying space" in _any_ Okio primitive. What does it mean for `Buffer()` to return from `awaitSpace(100500)`?